### DISCUSSION TITLE
Changing facial max recognition distance requirements

### DISCUSSION THREAD

**Discussion by VenomousViper27** [immich-app/immich#5654] [Q&A] [answered]
Does changing the facial max recognition distance require that the "recognize faces" job be completely re-run? Or will the setting change just take place on new assets?

I had previously lowered the value (to 0.6) and ran a full scan, but I'd like to set it back to default 0.7 now that I've fixed a lot of the overlapped detected faces and I'm seeing faces that already exist not matching the existing people that it should. Thank you!

**Comment by mertalev** [comment, marked as answer]
Hi, you don't need to re-run facial recognition after changing this value, but the change is only applied to later jobs (either adding new assets or re-running facial recognition). 

That being said, I think the default max distance is 0.6. You might be thinking of the min face detection threshold that defaults to 0.7. The same applies for this setting too.

**Reply by VenomousViper27** [reply]
Whew! That saves me from redoing some work. Thank you!
